Makefile

```diff
@@ -309,3 +308,9 @@ init:
 		echo "Please download and install $(REQUIRED_GO_VERSION) using brew or other method of your choice"; \
 		exit 1; \
 	fi
+
+smoke-test:
+	go test -short -coverprofile smoke_coverage.txt -failfast ./awscsm/... ./cfg/... ./cmd/... ./handlers/... ./internal/... ./logger/... ./logs/... ./metric/... ./plugins/... ./profiler/... ./tool/... ./translator/...
```

Comment:
`smoke_coverage.txt` is generated on every run, right? is it worth adding to the .gitignore?

Reply:
Good idea. I'll add it